tests/test_rocket.py

```diff
@@ -76,3 +77,178 @@ def mainTrigger(p, y):
     )
 
     assert test_rocket.allInfo() == None
+
+
+def test_evaluate_static_margin_assert_cp_equals_cm(kg, m, dimensionless_rocket):
+    rocket = dimensionless_rocket
+    rocket.evaluateStaticMargin()
+
+    assert rocket.centerOfMass(0) / (2 * rocket.radius) == rocket.staticMargin(0)
+    assert rocket.centerOfMass(-1) / (2 * rocket.radius) == rocket.staticMargin(-1)
```

Comment:
I am not sure that the intended behavior is happening here. I assume (you may add some comments in the code to make it clearer) that what you are trying to do is get the last value, in time, of the center of mass and of the static margin. This would mean evaluating both at `rocket.motor.burnOutTime` instead of at `-1` (which would work in a list or a numpy array).

So, there are two implementation options:

```python
    burnOutTime = rocket.motor.burnOutTime
    assert rocket.centerOfMass(burnOutTime) / (2 * rocket.radius) == rocket.staticMargin(burnOutTime)
```

or, if both `rocket.centerOfMass` and `rocket.staticMargin` have a numpy array or list as a source (which I am not sure is guaranteed):

```python
    assert rocket.centerOfMass[-1, 1] / (2 * rocket.radius) == rocket.staticMargin[-1, 1] # [-1, 1] =  last row, second column 
```

This use of (-1), which I believe is mistaken (but I maybe wrong), happens 4 times in this code.